stonesoup/metricgenerator/tracktotruthmetrics.py

```diff
@@ -277,9 +288,14 @@ def LS(self, manager):
                           for truth in manager.groundtruth_paths)
 
         timestamps = manager.list_timestamps()
+        try:
+            LS = numerator / denominator
+        except ZeroDivisionError:
+            # No truth
+            LS = 0
         return TimeRangeMetric(
             title="SIAP LS",
-            value=numerator / denominator,
+            value=LS,
```

Comment:
Returning Zero here is more reasonable (0 = worst performance), however the metric demoninator is is the sum of the total time for which there is a true object in the scene. Therefore a 0 indicates that there is no true objects within the data being evaluated so again an error message is probably more appropriate.